src/test/java/io/cdap/plugin/ariba/source/AribaRecordReaderTest.java

```diff
@@ -71,10 +71,12 @@ public void testInitialize() throws IOException, AribaException, InterruptedExce
       }
     };
     aribaRecordReader.initialize(aribaInputSplit, null);
-    aribaRecordReader.getProgress();
+    aribaRecordReader.nextKeyValue();
+    String progress = String.valueOf(aribaRecordReader.getProgress());
     aribaRecordReader.close();
-    Assert.assertEquals(aribaRecordReader.getCurrentKey(), NullWritable.get());
+    Assert.assertNull(aribaRecordReader.getCurrentValue().get("IsTestProject"));
```

Comment:
why is the value null?

Reply:
This test case primarily test for initialize() method.

Reply:
Fixed

Added StructuredRecord and validated value instead of null based on that.